### PR TITLE
Cookbook cleanup

### PR DESCRIPTION
**Pull request by tas50**

### CODE REVIEW

File: .delivery/project.toml [hunk 1]
```diff
@@ -0,0 +1 @@
+remote_file = "https://raw.githubusercontent.com/chef-cookbooks/community_cookbook_tools/master/delivery/project.toml"
```
Comment (MarkGibbons) [inline .delivery/project.toml]: Pulling in an rspec test but no spec directory exists.

File: .kitchen.yml [hunk 2]
```diff
@@ -3,21 +3,23 @@ driver:
...
-  - name: opensuse-13.2
-  - name: opensuse-leap-42.1
+  - name: fedora-27
+  - name: opensuse-leap-42
   - name: ubuntu-14.04
     run_list: apt::default
   - name: ubuntu-16.04
     run_list: apt::default
+  - name: ubuntu-18.04
+    run_list: apt::default
```
Comment (MarkGibbons) [inline .kitchen.yml]: doesn't exist yet, probably be there soon?

File: .delivery/project.toml (hunk 1 above)
Comment (tas50) [reply .delivery/project.toml]: Added

File: .kitchen.yml (hunk 2 above)
Comment (tas50) [reply .kitchen.yml]: It's up now

**Pull request merged**